watsapp/app.py

```diff
@@ -0,0 +1,68 @@
+import json, os, logging
+from dateutil import tz
+from config import *
```

Comment:
`from config import *` is a bad practice. 

If you follow the principle explicit is better than implicit, the proper way is to import every specific function or class that you are using in the file